charters/DataPortal/charter.md

```diff
@@ -0,0 +1,69 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data.
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyze, and make available the data in the Human Cell Atlas project.
+
+*Data Browser* A DCP application which provides a faceted search capability.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for users to search, contribute to, or learn about, the HCA genetic data in order to identify a subset of the data for download and/or further analysis at petabyte scale on public clouds.
+
+## In-scope
+
+### Core capabilities
+* Implement a UX, visual, and branding design with input from the HCA brand working group.
```

Comment:
Is UX the right term here? do you mean UI?

Reply:
>Implement a UX, visual, and branding design with input from the HCA brand working group.

I would probably say here, in two separate points:
 - Implement a visual design which addresses the needs and goals of the identified main user personas. 
It should therefore be driven by the user experience research insights and interaction design guidelines coming from the User Experience (UX) team. 
 - Implement a visual design which is in line with the Human Cell Atlas brand guidelines.

Reply:
Updated the first two Core capabilities. Does this look OK now?